tests/deployment/vlan_networks.py

```diff
@@ -146,7 +155,7 @@ def delete_networks():
             # Assert that none of the nodes are on any network
             port_networks = self.get_port_networks(ports)
             for node in nodes:
-                assert self.get_network(node.nics[0].port, port_networks) == set()
+                assert self.get_network(node.nics[0].port, port_networks) == set()  # noqa
```

Comment:
I would like to do that but `port_networks` is a solid name for it.  We could do something like `port_nets` but I'm not a huge fan of that.